swift-sdk/IterableMessaging.swift

```diff
@@ -43,6 +43,12 @@ import Foundation
     /// - parameter source: The source of deletion `inboxSwipe` or `deleteButton`.`
     @objc(removeMessage:location:source:) func remove(message: IterableInAppMessage, location: InAppLocation, source: InAppDeleteSource)
     
+    /// - parameter message: The message to remove.
+    /// - parameter location: The location from where this message was shown. `inbox` or `inApp`.
+    /// - parameter source: The source of deletion `inboxSwipe` or `deleteButton`.`
+    /// - parameter inboxSessionId: The ID of the inbox session that the message originates from.
+    @objc(removeMessage:location:source:inboxSessionId:) func remove(message: IterableInAppMessage, location: InAppLocation, source: InAppDeleteSource, inboxSessionId: String?)
+    
```

Comment:
Not needed for this PR. But I think we should take another look at public methods and classes. I think we don't need to expose inboxSessionId publicly. Since this will never be set by developers. But that is for another Jira issue and PR.

Reply:
Created a ticket and also put this specific function in there, because I agree, this probably does not need to be public.